### PR TITLE
Mccalluc/isort everywhere

### PR DESCRIPTION
**Pull request by mccalluc**
Extending the work from #1831.

This is the result of `isort -rc .`, but
- excluding migrations
- reverting a comment deletion in `refinery/config/settings/aws.py` (I think it interprets a comment in the imports as a section header, and we do not have those turned on.)

Questions:
- Some packages we often put in the django section, and perhaps they should be kept there? `mock`, `mockcache`, `django_extensions`
- Are you happy with the handling of `from` vs `import`?
- Should migrations be explicitly excluded here? Or should they be re-ordered, too? Or just ignore them?
- If folks preferred a different style for the long lists of imports, that could be tweaked.

I don't have strong opinions about any of these. I do think we should wait till the start of the next release before merging this.

### CODE REVIEW

File: refinery/analysis_manager/tasks.py [hunk 1]
```diff
@@ -6,20 +6,20 @@
 import logging
 import urlparse
 
-from bioblend import galaxy
 import celery
+import requests
```
Comment (hackdna) [inline refinery/analysis_manager/tasks.py]: Why is `requests` before `celery.result`?

File: refinery/analysis_manager/tasks.py [hunk 2]
```diff
@@ -6,20 +6,20 @@
...
+from bioblend import galaxy
```
Comment (hackdna) [inline refinery/analysis_manager/tasks.py]: Why is `bioblend` after `celery`?

File: refinery/analysis_manager/utils.py [hunk 3]
```diff
@@ -4,11 +4,12 @@
 import os
 import re
 
-import requests
 from django.conf import settings
 from django.contrib.auth.models import User
 from django.http import HttpResponseServerError
 from django.utils import timezone
+
+import requests
```
Comment (hackdna) [inline refinery/analysis_manager/utils.py]: Why is `requests` before `jsonschema`?

File: refinery/analysis_manager/tests.py [hunk 4]
```diff
@@ -1,10 +1,11 @@
 import json
 import uuid
 
-import mock
 from django.contrib.auth.models import User
 from django.http import HttpResponseBadRequest, HttpResponseNotAllowed
 from django.test import RequestFactory, TestCase
+
+import mock
```
Comment (hackdna) [inline refinery/analysis_manager/tests.py]: Why is `mock` before `guardian`?

**Comment by hackdna** [comment]
The comment in the import section of `aws.py` is superfluous and can be removed. However, there appear to be multiple issues with ordering. I've commented on the first few that I found. Third-party packages like `mock` should be grouped with other third-party packages. Migration are generated automatically so should probably be ignored (they often don't even confirm to PEP8).

**Comment by mccalluc** [comment]
> However, there appear to be multiple issues with ordering. I've commented on the first few that I found.

The default behavior is to separate the different types of imports; I overridden that default, to both kinds of imports are sorted together.

> Third-party packages like mock should be grouped with other third-party packages.

So no action needed there.

> Migration are generated automatically so should probably be ignored (they often don't even confirm to PEP8).

Add an ignore to cover all migrations

**Comment by mccalluc** [comment]
Close in favor of #1927

**Pull request closed without merging**